test/stability/testMerge/testMerge.go

```diff
@@ -19,35 +18,50 @@ import (
 )
 
 // Test with testScenario=1:
-// This part of test generates packets on ports 0 and 1, receives packets
-// on 0 port. Packets generated on 0 port has IPv4 source addr ipv4addr1,
-// and those generated on 1 port has ipv4 source addr ipv4addr2. For each packet
-// sender calculates IPv4 and UDP checksums and verify it on packet receive.
-// When packet is received, hash is recomputed and checked if it is equal to value
-// in the packet. Test also calculates sent/received ratios, number of broken
-// packets and prints it when a predefined number of packets is received.
+// This part of test generates packets sends on 2 ports and receives packets
+// on 1 port (merged by second part). Checks quantity and contents of packets according to scenario.
 //
 // Test with testScenario=2:
-// This part of test receives packets on 0 and 1 ports, merges flows
-// and send result flow to 0 port.
+// This part of test receives packets on 2 ports, merges flows
+// and send result flow to 1 port. There are several scenarios:
+// 0: getGet - get packets, merge and send back
+// 1: inCopy - stop one of flows, copy the remaining, merge original flow with its copy and send back
+// 2: copyCopy - copy both flows, merge all 4 and send back
+// 3: segmSegm - add segment functions at both flows, merge and send
+// 4: segmGet - add segment function to one flow, merge and send
+// 5: segmCopy - stop one of flows, add segment function to the remaining, copy it , merge original flow with its copy and send back
+// 6: inSegm - stop one flow, split second, merge and send back
+// 7: manyFlows - split one flow, copy another and stop original, merge copy and two splitted, send
```

Comment:
Please rename and modify:
0 -> 0
---
4 -> 1
3 -> 2 multipleSegm
6 -> 3 singleSegm
---
1 -> 4 getCopy (merge also with get, don't stop it)
2 -> 5 
7 -> 6 segmCopy
5 - It is part of sequential function and really is identical with 2 from the point of merge. Redundant here.